app/src/main/res/values-am-rET/strings.xml

```diff
@@ -76,8 +76,8 @@
   <string name="patiententry_error_both_dateofbirth_and_age_empty">የታካሚውን ዕድሜ ያስገቡ</string>
```

Comment:
Can we squash this commit with the earlier one where we made the changes to the source strings?